src/ossf_scorecard/scorecard.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+def get_scorecard(platform, org, repo):
```

Comment:
We also need to check if these inputs are valid.

Reply:
yes I am checking this here

https://github.com/nexB/ScoreCode/blob/60a880f7182feb32843ba0b964c05987edd5eebe/src/ossf_scorecard/scorecard.py#L126

Reply:
Maybe we should also check if the URL exists, as an option or by default?

Reply:
Done 

https://github.com/nexB/ScoreCode/pull/5/commits/414e56d7e37dcc22013c3549e99ca3f12cc24291